src/TBuffer.cpp

```diff
@@ -703,10 +703,20 @@ TBuffer::TBuffer(Host* pH)
 , mIgnoreTag(false)
 , mSkip()
 , mParsingVar(false)
+, mOpenMainQuote()
 , mMXP_SEND_NO_REF_MODE(false)
+, mEchoText(false)
 , gotESC(false)
 , gotHeader(false)
 , codeRet(0)
+, mWaitingForHighColorCode(false)
```

Comment:
Did you do this re-ordering manually or automated? I found it to be a really slow process to do it manually.

Reply:
Manually - I use my wide-screen monitor(s) and show the initialisation list in one window and the header file in another which are placed side by side (*that is why I like all the members and methods segregated in the latter so the members are in single blocks {of visibility} rather than interspersed and scattered amongst methods that refer to them throughout the whole class declaration*)...